src/sdk.ts

```diff
@@ -726,7 +726,7 @@ export class OpenSeaSDK {
     const getConsiderationItemsFromSellerFees = (
       fees: Fees
     ): ConsiderationInputItem[] => {
-      const sellerFees = new Map(Object.entries(fees?.sellerFees));
+      const sellerFees = fees?.sellerFees as Map<string, number>;
```

Comment:
might be a dumb question but isn't `sellerFees` already a map according to the types? why do we need to coerce this here

Reply:
realizing now, i think alternative #2 in this pr description is required for that? https://github.com/ProjectOpenSea/opensea-js/pull/727 
this might be the better approach to avoid having to do the coercion anywhere else

Reply:
hmm, yeah I don't think we should be doing any manual casting for cleanliness

Reply:
sounds good, i'll update the pr to just set it properly in `collectionFromJSON`